content/en/docs/installation/openshift.md

```diff
@@ -62,10 +62,10 @@ are included in a single YAML manifest file:
 Install the `CustomResourceDefinitions` and cert-manager itself
 ```bash
 # OpenShift 4+
-oc apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.yaml
+oc apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```

Comment:
```suggestion
oc apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```

Reply:
Hmm, actually, I'm not sure about the `oc` tool. You know more about that than I do.